src/components/minionCancel.jsx

```diff
@@ -49,7 +56,7 @@ const MinionCancel = ({ proposal }) => {
           display: 'inline-block',
         }}
       >
-        <Button onClick={cancelMinion} disabled={!isCorrectChain}>
+        <Button onClick={cancelMinion} disabled={!isCorrectChain || !isMember}>
```

Comment:
wondering if this should be `(minionAction?.memberOnlyEnabled && !isMember)`?

Reply:
It should be `!isProposer` for this one I believe.